packages/plugin-commands-store/src/store.ts

```diff
@@ -94,6 +96,13 @@ export async function handler (opts: StoreCommandOptions, params: string[]) {
   }
 }
 
+async function pathCmd (opts: StoreCommandOptions) {
+  logger.info({
+    message: await storePath(opts.dir, opts.storeDir),
```

Comment:
just return the string. The CLI will print it. And it is easy to cover with a test.

Reply:
Oh nice, yes that does make things easier 👍